src/components/forms/InputField.tsx

```diff
@@ -104,13 +104,15 @@ export function InputField(props: Props) {
           )}
           placeholder={props.placeholder || ''}
           onBlur={(event) => {
-            event.target.value =
-              event.target.value === '' && props.type === 'number'
-                ? '0'
-                : event.target.value;
+            if (!props.changeOverride) {
```

Comment:
@beganovich We forgot to avoid unnecessarily triggering the onValueChange function if this prop isn't set to "true". So, basically the function was being called twice - I just added this check to avoid calling it onBlur.